targets/TARGET_STM/mbed_overrides.c

```diff
@@ -27,10 +27,69 @@
  */
 #include "cmsis.h"
 #include "objects.h"
+#include "platform/mbed_error.h"
 
 int mbed_sdk_inited = 0;
 extern void SetSysClock(void);
 
+#if defined(RCC_LSE_HIGHDRIVE_MODE) || defined(RCC_LSEDRIVE_HIGH)
+#   define LSE_CONFIG_AVAILABLE
+#endif
+
+// set defaults for LSE drive load level
+#if defined(LSE_CONFIG_AVAILABLE)
+
+#   if defined(MBED_CONF_TARGET_LSE_DRIVE_LOAD_LEVEL)
```

Comment:
I'm not entirely sure if something changed since [1] was documented, but I think the build system will interpret the "macro_name" from target.json _without_ adding the "MBED_CONF_" prefix, which will lead to the setting never being applied. I noticed this while I was backporting this to mbedos 5 so I might be wrong.

[1] https://os.mbed.com/docs/mbed-os/v6.11/program-setup/advanced-configuration.html

Reply:
Hi
I am sorry, I didn't understand well the question.
I don't think there is something different between mbedos 5 and 6 about json configuration.
And no need to add this line for each target as it can be defined at the family level.

Reply:
Sorry, I didn't made my point in an understandable way:
The implementation requires `MBED_CONF_TARGET_LSE_DRIVE_LOAD_LEVEL` to be defined to set LSEDRV from the config. However, in `target.json` the actual parameter is set to  `"macro_name": "TARGET_LSE_DRIVE_LOAD_LEVEL"` which will result in the target _not_ being generated as `MBED_CONF_TARGET_LSE_DRIVE_LOAD_LEVEL`, but without the prefix `MBED_CONF_`. (See [1] above comment)

This results in the value from `target.json` (or anywhere else, where the setting is overridden from) never being used in the first place, as far as I understand.

Reply:
I'm sorry, was resolved here https://github.com/ARMmbed/mbed-os/pull/13939
Never mind. My fault.